### PR TITLE
Minor improvements and fixes

### PR DESCRIPTION
**Pull request by ua741**
- Fix: Fetch 2FA status when account is configured
- Add delay in refreshing 2FA status on appStart

### CODE REVIEW

File: lib/ui/components/toggle_switch_widget.dart [hunk 1]
```diff
@@ -60,7 +60,7 @@ class _ToggleSwitchWidgetState extends State<ToggleSwitchWidget> {
               activeColor: enteColorScheme.primary400,
               inactiveTrackColor: enteColorScheme.fillMuted,
               materialTapTargetSize: MaterialTapTargetSize.shrinkWrap,
-              value: toggleValue,
+              value: toggleValue ?? false,
```
Comment (ashilkn) [inline lib/ui/components/toggle_switch_widget.dart:63]: `toggleValue` is non-nullable, why is this needed?

Comment (ua741) [reply lib/ui/components/toggle_switch_widget.dart:63]: I was seeing null values in errors. Does it depend on some package/method which is not null safe?

**Review by ashilkn — APPROVED** [review]
